packages/okta-angular/package.json

```diff
@@ -17,8 +17,9 @@
     "prepare": "yarn ngc",
     "prengc": "yarn build:package-info",
     "start": "yarn --cwd test/e2e/harness/ start",
-    "test": "yarn lint && yarn test:e2e && yarn test:unit",
-    "test:e2e": "yarn --cwd test/e2e/harness/ e2e",
+    "test": "yarn lint && yarn e2e && yarn test:unit",
+    "pree2e": "yarn build:dependencies",
+    "e2e": "yarn --cwd test/e2e/harness/ e2e",
```

Comment:
Any reason to change this from `test:e2e` to `e2e` throughout all of these package updates? By doing so, we're losing out on the naming convention of `test:${type}` laid out across this monorepo.

Reply:
I wanted to run `pree2e` script since we won't be able to run `yarn test:e2e` without first running `yarn pretest`.
The general convention in the yarn world I believe is to have `e2e` and `pree2e` scripts.
This is needed for running the e2e tests on bacon. 
I could instead do `yarn pretest; yarn test:e2e` but running `yarn e2e` alone seemed cleaner.